ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5665,3 +5665,124 @@ kong_apis:
       - name: request-size-limiting
         config.allowed_payload_size: "{{ medium_request_size_limit }}"
 
+  - name: updateUserV2
+      uris: "{{ user_service_prefix }}/v2/update"
+      upstream_url: "{{ learning_service_url }}/v2/user/update"
+      strip_uri: true
+      plugins:
+        - name: jwt
+        - name: cors
+        - "{{ statsd_pulgin }}"
+        - name: acl
+          config.whitelist:
+            - 'userUpdate'
+        - name: rate-limiting
+          config.policy: local
+          config.hour: "{{ medium_rate_limit_per_hour }}"
+          config.limit_by: credential
+        - name: request-size-limiting
+          config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: managedUserV1
```

Comment:
can we have meaningful api names? like managedUserV1Create or managedUserV1Update or managedUserV1Delete for all the below apis as well, the api names should indicate what the apis do @Hari-stackroute

Reply:
@SMYALTAMASH : yes , sreerag will be doing the changes.

Reply:
Done